README.md

```diff
@@ -233,13 +233,13 @@ The following options are valid inside a part configuration:
   **default**: The first entry in `values =`.
 
   If the value of the part matches this value it is considered optional, i.e.
-  it's representation in a `--serialize` possibility is not required.
+  its representation in a `--serialize` possibility is not required.
 
-  Example::
+  Example:
 
     [bumpversion]
     current_version = 1.alpha
-    parse = (?P<num>\d+)\.(?P<release>.*)
+    parse = (?P<num>\d+)(\.(?P<release>.*))?
```

Comment:
Making the `.` optional between `num` and `release` is necessary, because otherwise the code crashes when running `bump2version release` with `current_version = 1.beta`:
```
Evaluating 'parse' option: '(?P<num>\d+)\.(?P<release>.*)' does not parse current version '1'
Traceback (most recent call last):
  File "/Users/guillaumelostis/.venv/bump/bin/bump2version", line 11, in <module>
    load_entry_point('bump2version', 'console_scripts', 'bump2version')()
  File "/Users/guillaumelostis/code/bump2version/bumpversion/cli.py", line 118, in main
    context = _commit_to_vcs(files, context, config_file, config_file_exists, vcs, args, current_version, new_version)
  File "/Users/guillaumelostis/code/bump2version/bumpversion/cli.py", line 662, in _commit_to_vcs
    context.update({'new_' + part: new_version[part].value for part in new_version})
TypeError: 'NoneType' object is not iterable
```

Reply:
I'd prefer to keep that in a separate PR since this just mentions typos.

Reply:
Sure, let me remove that from this PR, and open a separate one with it!

Reply:
PR opened in #92